File: back-end/src/main/java/com/huyphan/dtos/CartItemDto.java
package com.huyphan.dtos;

import lombok.Getter;
import lombok.Setter;

/** Cart item DTO. */
@Getter
@Setter
public class CartItemDto {

	/** Book id of the book belongs to this cart item. */
	private int bookId;

	/** The amount of this cart item. */
	private int amount;

	/** The book of this cart. */
	private BookDto bookDto;
}

File: front-end/src/app/core/services/dtos/cart-item-dto.ts
import { BookDto } from './book-dto';

/** Cart item DTO. */
export interface CartItemDto {
  /** Id of the book belongs to this cart. */
  readonly bookId: number;

  /** The amount of this cart item. */
  readonly amount: number;

  /** The book of this cart. */
  readonly book: BookDto;
}
